src/Xamarin.Android.Build.Tasks/Utilities/Files.cs

```diff
@@ -198,16 +198,16 @@ public static void ExtractAll(ZipArchive zip, string destination, Action<int, in
 			int i = 0;
 			int total = zip.Entries.Count;
 			foreach (var entry in zip.Entries) {
-
-				if (string.Equals(entry.FullName, "__MACOSX", StringComparison.OrdinalIgnoreCase) ||
-						string.Equals(entry.FullName, ".DS_Store", StringComparison.OrdinalIgnoreCase))
+				if (entry.FullName.Contains ("__MACOSX") ||
```

Comment:
If `entry.FullName` *does* contain path information, is the directory separator char at all normalized? The problem with the updated logic is that it'll process `my-file.ds_store`, which doesn't seem *at all* correct. Perhaps this should instead check for `entry.FullName.EndsWith("/.DS_Store", StringComparison.OrdinalIgnoreCase)`?